interfaces/fiveg_core_gnb/v0/interface.yaml

```diff
@@ -0,0 +1,12 @@
+name: fiveg_core_gnb
+internal: true
+
+version: 0
+status: draft
+
+providers:
+  - name: sdcore-nms-k8s-operator
+    url: https://github.com/canonical/sdcore-nms-k8s-operator
+requirers: []
```

Comment:
I suggest fill up the the requirers, otherwise charm integration in the Charmhub becomes empty for the provider and requirer charms such as https://charmhub.io/sdcore-gnbsim-k8s/integrations?channel=latest/beta#fiveg_gnb_identity.